software/28.Gin/README.md

```diff
@@ -0,0 +1,139 @@
+# Workshop 2 - API REST in Go with Gin
+
+In this workshop we will learn how to use the main functions of the Gin package in Go and how to create a REST API.
+
+> :bulb: A quick [documentation](https://searchapparchitecture.techtarget.com/definition/RESTful-API) about REST API.
+
+## Step 0: Initialization
+
+All the required information to install the workshop's dependencies are given in the [SETUP.md](./SETUP.md)
+
+## Step 1: The codebase
```

Comment:
You need to add the template in this repo.

Reply:
which template ?

Reply:
The files that we need in the workshop.

Reply:
In the step 1 you talk about that you have designed a boilerplate for this project with folders but there is no folders in the repository. You have two options, the first is to indicate where create folders and after that talk about the project structure, the second option is to add theses folders directly in the repo.